source/sync/data-access-patterns/flexible-sync-permissions-guide.txt

```diff
@@ -0,0 +1,611 @@
+.. _flexible-sync-permissions-guide:
+
+===============================
+Flexible Sync Permissions Guide
+===============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+This page shows how to set up your flexible sync app's permissions for the
+following common use cases:
+
+- :ref:`read-write-own-data`
+- :ref:`write-own-data-read-all-data`
+- :ref:`admin-privileges`
+- :ref:`restricted-news-feed`
+- :ref:`dynamic-collaboration`
+- :ref:`tiered-privileges`
+
+These assume you have already created a Realm app.
+
+.. seealso::
+
+   - :ref:`create-realm-apps`
+   - :ref:`flexible-sync-rules-and-permissions`
```

Comment:
this list isn't rendering properly for me 🤔 

<img width="819" alt="Screen Shot 2022-05-27 at 1 28 12 PM" src="https://user-images.githubusercontent.com/90647379/170760883-c9effb3c-33a6-4d1e-9388-949edf68701c.png">

Reply:
Can you refresh and see how it looks? Looks right in my browser: 
![image](https://user-images.githubusercontent.com/32645888/170779933-d33cfd0e-4791-417c-9d6c-dd256bccb3c1.png)

Reply:
me too. just some silly snoot i guess